### DISCUSSION TITLE
DNSSEC DS RECORD for zone

### DISCUSSION THREAD

**Discussion by brdelphus** [TechnitiumSoftware/DnsServer#1124] [Q&A] [answered]
Hi. I am unable to create a DS record using @ (as name). I returns me an error
`Missing! Please enter a name for the DS record.`

is that correct ? Version 13.2 docker image

**Comment by ShreyasZare** [comment, marked as answer]
Thanks for asking. DS record can exists only in parent zone and thus you cannot create DS at the zone's apex.
